src/NLog/NewRelic.LogEnrichers.NLog/NewRelicJsonLayout.cs

```diff
@@ -60,7 +62,7 @@ internal NewRelicJsonLayout(Func<NewRelic.Api.Agent.IAgent> agentFactory) : base
                 ExcludeProperties = ExcludeProperties
             };
 
-            Attributes.Add(new JsonAttribute("Message Properties", _jsonLayoutForMessageProperties, false));
+            Attributes.Add(new JsonAttribute(UserPropertyKey, _jsonLayoutForMessageProperties, false));
```

Comment:
Just to doublecheck, NLog lets you place a sub-object and will build the properties with the parent name . child attribute name?   If so, coolz.

Reply:
Yep, that's how that works.